client/interview-controller.js

```diff
@@ -0,0 +1,72 @@
+// Description:
+//   A Hubot script which helps hrs to control interviews.
+//
+// Commands:
+//  begin group Huntflow Reloaded
+//    begin admin
+//      hubot удалить интервью - remove an interview from the list
+//    end admin
+//  end group
+//
+
+const routines = require('hubot-routines')
+const utils = require('./utils')
+
+const interviewsService = require('./interviews-service')
+
+const deleteCandidateRegExp = new RegExp(/(удалить интервью кандидата)\s(((([А-Яа-я]+)|([A-Za-z]+))\s?){2})\s*/i)
+
+module.exports = async (robot) => {
+  // Checking if server user email & password was specified
+  if (!utils.SERVER_USER_EMAIL || !utils.SERVER_USER_PASSWORD) {
+    routines.rave(robot, utils.MSG_AUTHORIZATION_DATA_NEEDED)
+    return
+  }
+
+  robot.respond(/(удалить интервью)\s*$/i, async (msg) => {
+    if (!await routines.isAdmin(robot, msg.message.user.name.toString())) {
+      msg.send(utils.MSG_PERMISSION_DENIED)
+      return
+    }
+
+    let message = utils.MSG_ERROR_TRY_AGAIN
```

Comment:
Let's use it only if retry can help. Otherwise would be reasonable just retrieve the new token pair.